src/blocktypes/installables.nix

```diff
@@ -22,8 +22,10 @@
       flake,
       fragment,
       fragmentRelPath,
+      target,
     }: [
-      (import ./actions/build.nix flake fragment)
+      (import ./actions/build.nix target)
+      # profile commands require a flake ref
```

Comment:
seems worthy of an upstream issue

Reply:
in what respect?

Reply:
like `nix bundle` and `nix build` it should probably be able to work on `drv` files all-together?

Reply:
`nix build` does work on derivations